SPECS/azure-init/generate-vendor-tarball.sh

```diff
@@ -3,28 +3,28 @@
 # This script is used to generate the vendor tarball for libazureinit
 
 # For generating tar of releases
-# SOURCE_VERSION="0.1.1"
-# SOURCE_VERSION_V="v${SOURCE_VERSION}"
-# SOURCE_URL="https://github.com/Azure/azure-init/archive/refs/tags/${SOURCE_VERSION_V}.tar.gz"
+SOURCE_VERSION="0.1.1"
+SOURCE_VERSION_V="v${SOURCE_VERSION}"
+SOURCE_URL="https://github.com/Azure/azure-init/archive/refs/tags/${SOURCE_VERSION_V}.tar.gz"
 
 # For generating tar of PRs and Forks
-FORK_OWNER="Azure"
-SOURCE_VERSION="7142bced859169553e8948497aa13df742aac1ff"
-SOURCE_URL="https://github.com/${FORK_OWNER}/azure-init/archive/${SOURCE_VERSION}.tar.gz"
+# FORK_OWNER="Azure"
+# SOURCE_VERSION="7142bced859169553e8948497aa13df742aac1ff"
+# SOURCE_URL="https://github.com/${FORK_OWNER}/azure-init/archive/${SOURCE_VERSION}.tar.gz"
 
 workdir=$(mktemp -d)
-# cp 0001-add-Azure-Linux-support.patch $workdir/
+cp 0001-add-Azure-Linux-support.patch $workdir/
 
 pushd $workdir
 wget $SOURCE_URL
-tar -xzvf "${SOURCE_VERSION}.tar.gz"
+tar -xzvf "${SOURCE_VERSION_V}.tar.gz"
 pushd "azure-init-${SOURCE_VERSION}"
-# patch -p1 < ../0001-add-Azure-Linux-support.patch
+patch -p1 < ../0001-add-Azure-Linux-support.patch
```

Comment:
Curious, why are we applying a patch file within this script instead of in the spec?

Reply:
I hvae a habit of applying patches to the source tarballs in case the patch makes a change to the list of vendored dependencies